Allow excluding modules when discovering (#65)
tfausak/cabal-gild: README.md

  This pragma searches for files with any of the following extensions: `*.chs`,
  `*.cpphs`, `*.gc`, `*.hs`, `*.hsc`, `*.hsig`, `*.lhs`, `*.lhsig`, `*.ly`,
  `*.x`, or `*.y`,

  Directories can be quoted if they contain spaces.

>   Discovered modules can be ignored by using the `--exclude=MODULE` option. For
>   example:
> 
>   ``` cabal
>   library
>     -- cabal-gild: discover source/library --exclude=Foo.Bar

Author: ```suggestion
  Discovered modules can be ignored by using the `--exclude=FILE` option. For
  example:

  ``` cabal
  library
    -- cabal-gild: discover source/library --exclude=source/library/Foo/Bar.hs
```
